core.c

```diff
@@ -496,6 +496,25 @@ int readbyte(FILE * f)
     return strtol(data, NULL, 16);
 }
 
+int load_raw(struct em8051 *aCPU, char *aFilename)
+{
+    FILE *f;
+    if (aFilename == 0 || aFilename[0] == 0)
+        return -1;
+    f = fopen(aFilename, "r");
```

Comment:
Please use mode "rb" instead of "r", otherwise windows targets will just load garbage.